src/recording/recordingmanager.cpp

```diff
@@ -81,8 +89,8 @@ void RecordingManager::slotSetRecording(bool recording) {
     }
 }
 
-void RecordingManager::slotToggleRecording(double v) {
-    if (v > 0) {
+void RecordingManager::slotToggleRecording(bool toggle) {
+    if (toggle) {
```

Comment:
Is this okay?
This was `double` before, and the conversion to `bool` works flawlessly, even when I set  `[Recording],toggle_recording 2` with a wrongly configured WPushButton.

Reply:
Hmm, I think I'd prefer to use the explicit double type and then use `bool toggle = static_cast<bool>(value)`

Reply:
alright, reverted to double.